src/components/segments/MeetupDate.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import moment from 'moment';
+
+
+class MeetupDate extends React.Component {
+
```

Comment:
@Dsenza are you familiar with the concept of [React PropTypes](https://facebook.github.io/react/docs/reusable-components.html)?

I would recommend we add them for components that take in props, so it would be something like:

```js
static propTypes = {
  loading: React.PropTypes.bool.isRequired,
  . . .
};
```